src/app/components/form-user-profile/form-user-profile.component.html

```diff
@@ -0,0 +1,47 @@
+<form action="" class="form-profile">
+  <div class="form-profile__title-wrap">
+    <h2 class="form-profile__title">Profile</h2>
+    <button mat-stroked-button color="accent" type="submit">Save changes</button>
+  </div>
+  <div class="form-profile__change-photo">
+    <div class="form-profile__change-photo-desc">
+      <h3 class="form-profile___change-photo-title">Avatar</h3>
+      <p>Photo must be link</p>
+      <p>
```

Comment:
don't use tag 'p' for markup. div is better to use here

Reply:
but it don't correct to semantic

Reply:
It would be semantics if it were an article. But this is just an instruction.